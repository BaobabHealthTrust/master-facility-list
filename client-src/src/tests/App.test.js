import React from "react";
import ReactDOM from "react-dom";
// import FacilityList from "../Facility/FacilityList";
import renderer from "react-test-renderer";
import Enzyme, {shallow} from "enzyme";

class Sample extends React.Component {
  render() {
    return <div>Hey there... you!</div>;
  }
}

const sampleFacility = {
  code: "CP020001",
  common: "Hyatt Inc",
  dateOpened: "Invalid date",
  district: "Chitipa",
  id: 1,
  name: "Msumbe Health Center",
  ownership: "Christian Health Association of Malawi (CHAM)",
  regulatoryStatus: "Registration cancelled"
};

it("renders landing page", () => {
  const component = renderer.create(<Sample />);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
