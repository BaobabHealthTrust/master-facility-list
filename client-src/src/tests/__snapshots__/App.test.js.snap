// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing page 1`] = `
<div>
  Hey there... you!
</div>
`;
